src/services/pegout-data.processor.ts

```diff
@@ -507,8 +539,13 @@ export class PegoutDataProcessor implements FilteredBridgeTransactionProcessor {
     return this.pegoutStatusDataService.set(pegout);
   }
```

Comment:
As the `getLoggerContextInformation` method is no longer returning a string this logs an undefined.